source/gui/settingsDialogs.py

```diff
@@ -3204,34 +3204,35 @@ class AddonStorePanel(SettingsPanel):
 
 	def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 		sHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
-		# Translators: This is a label for the automatic updates combo box in the Add-on Store Settings dialog.
-		automaticUpdatesLabelText = _("Automatic &updates:")
-		self.automaticUpdatesComboBox = sHelper.addLabeledControl(
-			automaticUpdatesLabelText,
+		# Translators: This is a label for the add-ons update combo box in the Add-on Store Settings dialog.
+		addonsUpdateLabelText = _("Add-ons &update:")
+		self.addonsUpdateComboBox = sHelper.addLabeledControl(
+			addonsUpdateLabelText,
 			wx.Choice,
-			choices=[mode.displayString for mode in AddonsAutomaticUpdate],
+			choices=[mode.displayString for mode in AddonsUpdateMode],
 		)
-		self.bindHelpEvent("AutomaticAddonUpdates", self.automaticUpdatesComboBox)
-		index = [x.value for x in AddonsAutomaticUpdate].index(config.conf["addonStore"]["automaticUpdates"])
-		self.automaticUpdatesComboBox.SetSelection(index)
+		self.bindHelpEvent("AddonsUpdateMode", self.addonsUpdateComboBox)
+		index = [x.value for x in AddonsUpdateMode].index(config.conf["addonStore"]["automaticUpdates"])
+		self.addonsUpdateComboBox.SetSelection(index)
 
-		self.defaultUpdateChannelComboBox = sHelper.addLabeledControl(
-			# Translators: This is the label for the default update channel combo box in the Add-on Store Settings dialog.
-			_("Default update &channel:"),
+		self.defaultUpdateCheckChannelComboBox = sHelper.addLabeledControl(
+			# Translators: This is the label for the default update check channel combo box in the Add-on Store Settings dialog.
+			_("Default update check &channel:"),
 			wx.Choice,
-			# The default update channel for a specific add-on (UpdateChannel.DEFAULT) refers to this channel,
+			# The default update check channel for a specific add-on (UpdateChannel.DEFAULT) refers to this channel,
 			# so it should be skipped.
 			choices=[
 				channel.displayString for channel in UpdateChannel if channel is not UpdateChannel.DEFAULT
 			],
 		)
-		self.bindHelpEvent("DefaultAddonUpdateChannel", self.defaultUpdateChannelComboBox)
+		self.bindHelpEvent("DefaultAddonUpdateChannel", self.defaultUpdateCheckChannelComboBox)
 		index = config.conf["addonStore"]["defaultUpdateChannel"]
-		self.defaultUpdateChannelComboBox.SetSelection(index)
+		self.defaultUpdateCheckChannelComboBox.SetSelection(index)
 
 		self.allowIncompatibleUpdates = sHelper.addItem(
-			# Translators: Mute other apps checkbox in settings
-			wx.CheckBox(self, label=_("Allow automatic updates to install incompatible add-ons")),
+			# Translators: This is the label for the "Allow incompatible add-ons when checking for updates" checkbox
+			# in the Add-on Store Settings dialog.
+			wx.CheckBox(self, label=_("Allow incompatible add-ons when checking for updates")),
```

Comment:
this change doesn't make sense. the original ensures it is clear that you can update to incompatible add-ons. this change makes implications about the installed add-on being incompatible